pages/[id].js

```diff
@@ -110,18 +134,57 @@ export default function PlantDetails({ plants, onDeletePlant, onEditPlant }) {
   );
 }
 
-const Container = styled.article`
-  border: 2px solid black;
-  border-radius: 8px;
-  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
+const CardDetails = styled.article`
   padding: 10px 10px 30px;
   margin: 20px 38px 23px 35px;
+
+  @media (min-width: 720px) {
+    width: 50%;
+    margin: 20px auto 0 auto;
+  }
+`;
+
+const ButtonEdit = styled.button`
+  background-color: var(--color-button-edit);
+  margin: 5px;
+
+  &:hover {
+    background-color: var(--color-button-edit-hover);
+  }
+`;
+
+const ButtonDelete = styled.button`
```

Comment:
You used margin for the other buttons but not for the `ButtonDelete`. Is it intended? Otherwise your buttons lack a consistent design system.